### ISSUE TITLE
Unable to match a Python step that starts with a parameter

### ISSUE THREAD

**Issue by ethall** [alexkrechik/VSCucumberAutoComplete#242]
**Describe the bug**
The document validator is unable to match a Gherkin step line if that line's corresponding definition starts with a parameter.
I've only tested this against a step definition implemented in Python.

Autocomplete has no problem suggesting a step definition that starts with a parameter.

**Expected behavior**
The extension should be able to successfully match step lines to step definitions that begin with parameters.

**Cucumberautocomplete part of VSCode settings:**
```
"cucumberautocomplete.steps": [
    "tests/steps/*.py"
],
"cucumberautocomplete.syncfeatures": "tests/features/*.feature"
```

**Step definition**
The following example is for a `then` step, but the bug applies to all step types.
```python
@then('{device} is in the {state_type} state')
```

**Gherkin step line**
The following example is for a `then` step, but the bug applies to all step types.
```gherkin
Then USB1 is in the OFF state
```

**Comment by ethall** [comment]
I tried the `cucumberautocomplete.customParameters` regex from #197 with the following addition to my settings:
```
"cucumberautocomplete.customParameters": [
    {
        "parameter": "/\\{.*\\}/",
        "value": ".*"
    }
],
```
but it had no effect.
However, explicitly specifying the parameter allows it to be matched:
```
"cucumberautocomplete.customParameters": [
    {
        "parameter": "{device}",
        "value": ".*"
    }
],
```
Though this is an inconvenient workaround, since all future parameters must be specified.

**Comment by alexkrechik** [comment]
Should be fixed in 2.14.0
